add servicenow as a metadata source (#19457)
DataDog/documentation: content/en/tracing/service_catalog/setup.md

- `metadata.description` gets mapped to `description`
- Other `specs` values get mapped to custom tags

## Import data from ServiceNow

> You can populate your Datadog Service Catalog with services from your ServiceNow CMDB with Service Ingestion feature in ServiceNow integration. Learn about how to start [here][16].

Reviewer: ```suggestion
You can populate your Datadog Service Catalog with services from your ServiceNow CMDB by using the Service Ingestion feature in the [Datadog-ServiceNow integration][16].
```
